tests/main/helpers/test_search_helpers.py

```diff
@@ -182,7 +182,6 @@ def test_clean_request_args(self):
             'question1': 'true',
             'question2': 'true',
             'question3': ['option1', 'option2'],
-            'page': 'false',
```

Comment:
Is this a flake8 related change?

Reply:
It is indeed. Apologies - updated the PR with the flake8 report after changing to the new extension. It reported this: `./tests/main/helpers/test_search_helpers.py:185:13: F601 dictionary key 'page' repeated with different values`

Reply:
Aha! Cool, thanks for adding the report.